build.sbt

```diff
@@ -591,19 +593,11 @@ lazy val fatjarPackageSettings = List[Def.Setting[_]](
       oldStrategy(x)
   },
   (Compile / Keys.`package`) := {
-    val slimJar = (Compile / Keys.`package`).value
-    val fatJar = crossTarget.value / (assembly / assemblyJarName).value
-    val _ = assembly.value
-    IO.copyFile(fatJar, slimJar, CopyOptions().withOverwrite(true))
-    slimJar
+    assembly.value
```

Comment:
I squinted hard at this and I feel like copying is not necessary, as the actual path to the jar doesn't matter.

My hunch is that for some reason copying causes issues (https://github.com/sbt/io/issues/371) which result in flaky CI.

Tests all seem green as well, so I doubt there's a change in published artifacts. Will check the snapshot deployment on maven central to confirm jars are still usable.